apps/wpcom-block-editor/src/wpcom/features/tracking/wpcom-block-premium-content-stripe-connect.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Internal dependencies
+ */
+import tracksRecordEvent from './track-record-event';
+
+/**
+ * Return the event definition object to track `wpcom_block_premium_content_upgrade_click`.
```

Comment:
Minor copy/paste typo:

```suggestion
 * Return the event definition object to track `wpcom_block_premium_content_stripe_connect_click `.
```